source/qml/subsystems/smog1/ObcSubsystem.qml

```diff
@@ -252,6 +252,12 @@ Subsystem {
             description: "TCXO works"
             content: telemetry4packet.diag.status.tcxoWorks ? "OK" : "NO"
         },
+        // Beacon
+        TelemetryObject {
+            group: "Beacon"
+            description: "Message"
+            content: String(telemetry4packet.message)
+        },
```

Comment:
Nice. I think I overlooked this in the past. Is it also added to the packet display?